lib/command/winRate.js

```diff
@@ -94,27 +95,44 @@ function twrFn(msg, authorRecord, tankName) {
 }
 
 function wrFn(msg, record) {
-	return this.wotblitz.account.info(record.account_id, null, null, ['statistics.all.battles', 'statistics.all.wins'])
+	return this.wotblitz.account.info(record.account_id, null, null, ['statistics.all.battles', 'statistics.all.wins', 'statistics.all.damage_dealt'])
 		.then(info => {
 			const wins = info[record.account_id].statistics.all.wins;
 			const battles = info[record.account_id].statistics.all.battles;
+			const damage = info[record.account_id].statistics.all.damage_dealt;
 			const result = {};
 
 			let percent = (wins / battles) * 100;
-			let send = `You have won ${wins} of ${battles} battles. That is ${percent.toFixed(2)}% victory!`;
+			let avgDmg = damage / battles;
+			let send = `You have won ${wins} of ${battles} battles. That is ${percent.toFixed(2)}% victory! Your average damage is ${avgDmg.toFixed(2)}.`;
 
 			if (record.wins && record.battles && battles - record.battles > 0) {
 				percent = (record.wins / record.battles) * 100;
-				send += `\nLast time you asked was ${battles - record.battles} battles ago, at ${percent.toFixed(2)}% victory.`;
+				send += `\nLast time you asked was ${battles - record.battles} battles ago, at ${percent.toFixed(2)}% victory`;
+
+				// Only print out previous damage if we've recorded it
+				if (record.damage) {
+					avgDmg = record.damage / record.battles;
+					send += ` and ${avgDmg.toFixed(2)} average damage dealt.`;
+				} else {
+					send += `.`;
+				}
 
 				percent = ((wins - record.wins) / (battles - record.battles)) * 100;
-				send += `\nOver those ${battles - record.battles} battles, you won ${percent.toFixed(2)}%!`;
+				avgDmg = ((damage - record.damage) / (battles - record.battles));
+				send += `\nOver those ${battles - record.battles} battles, you won ${percent.toFixed(2)}%`;
+
+				if (avgDmg) {
+					send += ` with average damage of ${avgDmg.toFixed(2)}`;
+				}
+				send += `!`;
```

Comment:
The punctuation was in the else block at line 118. Pick one pattern and use it in both places.